src/components/Header.jsx

```diff
@@ -1,108 +1,126 @@
-export default function Header({ user, theme, setTheme }) {
-    const handleCheckChange = () => {
-      if(theme === 'dark') {
-        setTheme('light');
-      } else {
-        setTheme('dark');
-      }
-    }
+import { useContext } from "react";
+import { Something, themeContext } from "../App";
 
-    const handleButtonClick = () => {
-      console.log("CLICK!");
+export default function Header() {
+  const { user } = useContext(Something);
+  const { theme, setTheme } = useContext(themeContext);
+  const handleCheckChange = () => {
+    localStorage.setItem("theme", "dark");
```

Comment:
You could write this more efficiently. You've currently got 3 calls to `localStorage`. You could instead write this like so:
```js
const handleCheckChange = () => {
   const newTheme = theme === "light" ? "dark" : "light"
   localStorage.setItem("theme", newTheme)
}
```